_layouts/base.html

```diff
@@ -0,0 +1,87 @@
+<!doctype html>
+<html lang="en-US">
+<head>
+
+    <meta charset="utf-8">
+    <!-- <meta http-equiv="X-UA-Compatible" content="IE=edge"> -->
+    <title>HackCU Slack Signup Page | 2018</title>
+    <!-- <link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon"> -->
+    <!-- <link rel="icon" href="img/favicon.ico" type="image/x-icon"> -->
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- for Google -->
+    <meta name="description" content="Join us for Local Hack Day at the University of Colorado Boulder."/>
+    <meta name="keywords" content="Hackathon, MLH, LocalHackDay, HackCU, Boulder, Colorado, CU, University"/>
+    <meta name="author" content="HackCU"/>
+    <meta name="theme-color" content="#315B7F">
+    <meta name="copyright" content="HackCU"/>
+    <meta name="application-name" content="Local Hack Day 2018"/>
+
+    <!-- for Facebook -->
+    <meta property="og:title" content="Local Hack Day 2018"/>
+    <!-- <meta property="og:image" content="img/og-image.png"/> -->
+    <!-- <meta property="og:image:secure_url" content="img/og-image.png"> -->
+    <meta property="og:url" content="https://local.hackcu.org"/>
+    <meta property="og:description" content="Join us for Local Hack Day at the University of Colorado Boulder."/>
+
+    <!-- for Twitter -->
+    <meta name="twitter:card" content="summary_large_image"/>
+    <meta name="twitter:title" content="Local Hack Day 2018"/>
+    <meta name="twitter:site" content="@hackcu">
+    <meta name="twitter:creator" content="@hackcu">
+    <meta name="twitter:description" content="Join us for Local Hack Day at the University of Colorado Boulder."/>
+    <meta name="twitter:image" content="https://local.hackcu.org/img/og-image.png"/>
+
+    <!-- Libs and Plugins CSS -->
+    <!-- <link rel="stylesheet" href="inc/bootstrap/css/bootstrap.min.css">
+    <link rel="stylesheet" href="inc/font-awesome/css/font-awesome.min.css"> -->
+     <!-- Font Icons -->
```

Comment:
Would be cool to use Biko (official HackCU font). See files: https://github.com/HackCU/HackCU/tree/master/assets/fonts

Reply:
👍